cmd/createcluster.go

```diff
@@ -672,17 +687,33 @@ func getValidators(
 			return nil, errors.Wrap(err, "builder registration to cluster object")
 		}
 
+		var partialDepositData []cluster.DepositData
+
+		// The loop over partial amounts to collect PartialDepositData
+		for i := range depositDatas {
+			for j, dd := range depositDatas[i] {
+				if [48]byte(dd.PublicKey) != dv {
```

Comment:
Should we error here if we never find a pubkey == dv?

Reply:
Fixed